app/api/v2/handlers/objective_api.py

```diff
@@ -32,9 +32,19 @@ async def get_objectives(self, request: web.Request):
         objectives = await self.get_all_objects(request)
         return web.json_response(objectives)
 
-    @aiohttp_apispec.docs(tags=['objectives'])
+    @aiohttp_apispec.docs(tags=['objectives'],
+                          summary='Retrieve objective by ID',
+                          description='Retrieve one objective by ID. Use fields from the `ObjectiveSchema` in '
+                                      'the request body to filter retrieved objective.',
+                          parameters=[{
+                            'in': 'path',
+                            'name': 'id',
+                            'schema': {'type': 'string'},
+                            'required': 'true',
+                            'description': 'Return UUID of the objective to be retrieved'}])
```

Comment:
You can remove "Return" here - the URL parameter just specifies which objective to return